plugins/sellStrategy.go

```diff
@@ -39,6 +39,7 @@ func makeSellStrategy(
 	config *sellConfig,
 ) (api.Strategy, error) {
 	pf, e := MakeFeedPair(
+		sdex,
```

Comment:
all these changes would be reverted once we add the `sdex` instance to `priceFeed.go`